README.md

```diff
@@ -39,6 +41,121 @@ The test suite uses the file `config/{NODE_ENV}.json` to override the settings i
 
 For more information see this [documentation](https://github.com/lorenwest/node-config/wiki/Environment-Variables#node_env).
 
+### Configuration for `sellers` within `./config/{NODE_ENV}`
```

Comment:
This is nearly identical to the section that was in Integration Tests' README with some additional input from the section that was in Broker Microservice's READE (there was some duplicated docs there)

I've moved it here to reflect the fact that `sellers`' config is not in either the `broker` nor `integrationTests` part of the config file